deployment/azuredeploy.json

```diff
@@ -49,6 +49,7 @@
         "staticWebsite": "[toLower(concat('learnclient', replace(variables('uniqueIdentifier'),'-','')))]",
         "keyVaultName": "[toLower(concat('kv-', variables('uniqueIdentifier')))]",
         "StorageAccountSuffix": "[toLower(concat(variables('uniqueIdentifier')))]",
+        "AuditLogsStorageAccountName": "[toLower(concat('auditlogs' , variables('StorageAccountSuffix')))]",

```

Comment:
Should we somehow indicate that these are KeyVault Audit logs 🤔 ?
> Maybe we can consider using `KeyVaultName` as second param of concat instead of `StorageAccountSuffix`, so that the 2 gets correlated.